src/epic.actions.js

```diff
@@ -29,7 +34,12 @@ async function describeEpic({ id }) {
 
 async function statusEpic({ id }) {
```

Comment:
can you just make `statusEpic` call `describeEpic` since the logic is basically all the same:

```js
async function statusEpic(opts) {
  const { epic } = await describeEpic(opts);
  return epic;
}
```

Reply:
that said, do we need 2 different functions here, or does destructuring give us enough. if they're going to diverge more, I'd say 2 functions, if not, then just one is fine and callers can throw away the extra data they don't need just as well as this function can.

Reply:
We need 2 functions here, as the destructuring does not give enough.